src/JsonSchema/Uri/UriResolver.php

```diff
@@ -28,7 +28,7 @@ class UriResolver implements UriResolverInterface
      */
     public function parse($uri)
     {
-        preg_match('|^(([^:/?#]+):)?(//([^/?#]*))?([^?#]*)(\?([^#]*))?(#(.*))?|', $uri, $match);
+        preg_match('|^(([^:/?#]+):)?(//([^/?#]*))?([^?#]*)(\?([^#]*))?(#(.*))?|', $uri ?: '', $match);
```

Comment:
This to me looks it would be better to add a string type hint to the uri parameter. But I'm not sure if it would cause further problems in caller code.

Reply:
Thanks this helps. I’ll take a look once I’m back from my holidays. Maybe an explicit string cast could help as well with the benefit of not breaking the function signature (yet).

Reply:
This has been done, once the pipeline is green I'll go ahead and merge it.

@Seldaek in the meantime if you can find the time (assuming you're having permissions) could you take a look a the branch protection and change it so that only `master`, `5.x.x` and `6.0.x` have the branch protection rules? I think two inclusion patterns (`master` and `[1-9].*`) would do the trick?
This would help with the rebasing of feature branches which I tend to do to keep a clean history in the feature branch.

Reply:
Ah sorry about that, fixed it now 👍🏻